app/controllers/actions/JourneyRefiner.scala

```diff
@@ -36,7 +36,6 @@ class JourneyRefiner @Inject()(cacheRepository: CacheRepository)(implicit val ex
       case Some(answers: Answers) if types.isEmpty || types.contains(answers.`type`) =>
         Right(JourneyRequest(answers, request))
       case a =>
```

Comment:
`case a` is necessary? ``Probably `case _ => ` will be better

Reply:
I mean previous implementation :)

Reply:
Sorry - I just saw the print statement.   Fixed now